frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -18,6 +18,7 @@ export const dashboardLogic = kea({
         renameDashboard: true,
         renameDashboardItem: (id) => ({ id }),
         renameDashboardItemSuccess: (item) => ({ item }),
+        setIsSharedDashboard: (id, is_shared) => ({ id, is_shared }),
```

Comment:
We should stick to `camelCase` in internal JS variables... even if the variable is going to be used by an API that requires it to be in `snake_case`. Perhaps some other reducer/listener will use it later?